File: app/src/main/java/sang/thai/tran/travelcompanion/model/Response.java
package sang.thai.tran.travelcompanion.model;

import com.google.gson.annotations.SerializedName;

public class Response {
    String Version;

    public int getStatusCode() {
        return StatusCode;
    }

    public void setStatusCode(int statusCode) {
        StatusCode = statusCode;
    }

    int StatusCode;

    public String getMessage() {
        return Message;
    }

    String Message;

    public Result getResult() {
        return result;
    }

    @SerializedName("Result")
    Result result;
}

File: app/src/main/java/sang/thai/tran/travelcompanion/model/Response.kt
package sang.thai.tran.travelcompanion.model

import com.google.gson.annotations.SerializedName

class Response {
    internal var Version: String? = null

    @SerializedName("StatusCode")
    var statusCode: Int = 0

    @SerializedName("Message")
    var message: String? = null
        internal set

    @SerializedName("Result")
    var result: Result? = null
        internal set
}
